app/views/accounts/show.html.slim

```diff
@@ -26,66 +27,69 @@
-
+          h3.title.is-4 History
+      .card-content.px-1
+        div.table-container
+          table.table.is-fullwidth.is-striped
+            thead
+              tr
+                th Date
+                th From
+                th To
+                th Amount
+                th Description
+                th Actions
+            tbody
+              = render partial: 'transaction', collection: account.transactions, account: account
 .columns
   .column
     .card
       .card-header
         .card-header-title.is-centered
-          h3.title Objectives
-      .card-content
-        table.table.is-fullwidth.is-striped
-          thead
-            tr
-              th Name
-              th Amount
-              th Progress
-              th Weeks # to goal
-              th Image
-              th Actions
-          = render partial: 'objective', collection: account.objectives, account: account
-      - unless public_share?(account)
-        .card-footer
-          = link_to '+Add', new_account_objective_path(account), class: 'card-footer-item'
-
+          h3.title.is-4 Objectives
+      .card-content.px-1
+        div.table-container
+          table.table.is-fullwidth.is-striped
+            thead
+              tr
+                th Name
+                th Amount
+                th Progress
+                th Weeks # to goal
+                th Image
+                th Actions
+            tbody
+              = render partial: 'objective', collection: account.objectives, account: account
+        - unless public_share?(account)
+          .card-footer
+            = link_to '+Add', new_account_objective_path(account), class: 'card-footer-item'
 .columns
   .column
     .card
       .card-header
-        .card-header-title
-          .title Weekly Automatic Topups
-      .card-content
-        table.table.is-fullwidth.is-striped
-          thead
-            tr
-              th From
-              th Amount
-              th Actions
-          - account.automatic_topup_configs.each do |config|
-            tr
-              td= config.from_account.name
-              td= config.amount
-              - if public_share?(account)
-                td ''
-              - else
-                td
-                  = link_to 'edit', edit_account_account_automatic_topup_config_path(account, config)
-                  | &nbsp;
-                  = link_to 'delete', account_account_automatic_topup_config_path(account, config), method: :delete, data: { confirm: 'Are you sure?' }
-      - unless public_share?(account)
-        .card-footer
-          = link_to '+ Add', new_account_account_automatic_topup_config_path(account), class: 'card-footer-item'
+        .card-header-title.is-centered
+          .title.is-4 Weekly Automatic Topups
+      .card-content.px-1
+        div.table-container
+          table.table.is-fullwidth.is-striped
+            thead
+              tr
```

Comment:
Perhaps we can add the `.has-text-centered` class?
What do you think about this?

Reply:
Ok, picked it up, but not implemented.